client/src/components/common/names-capture.vue

```diff
@@ -3,7 +3,7 @@
     <v-container fluid class="pa-0" id="send-to-examination-container">
       <template v-if="editMode">
         <v-row justify="end" class="mt-n3 mb-n5">
-          <v-col cols="6" class="copy-normal">I need a name to:</v-col>
+          <v-col cols="6" class="copy-normal copy-bold">I need a name to:</v-col>
```

Comment:
very minor thing, but the `.copy-bold` class already uses `@extend .copy-normal` so  I don't think we need to put both classes on the same element

Reply:
ohh right! I see, good catch. Will update. copy-normal isn't needed